vm/ubpf_jit_x86_64.h

```diff
@@ -348,4 +357,16 @@ emit_jmp(struct jit_state *state, uint32_t target_pc)
     emit_jump_offset(state, target_pc);
 }
 
+static inline void
+emit_string_load(struct jit_state *state, int dst, int string_id)
+{
+    emit_basic_rex(state, 1, RIP, dst);
+    emit1(state, 0x8d);
+    emit_modrm(state, 0, dst, RIP);
+    emit4(state, 0);
+    state->strings[state->num_strings].offset_loc = state->offset;
+    state->strings[state->num_strings].string_id = string_id;
+    state->num_strings++;
```

Comment:
potential array out-of-bound access. Can you please fix this and similar issues elsewhere?